### PR TITLE
DEV-28595 Paginated requests include all query params in link section

### PR DESCRIPTION
**Pull request by mrnyax**
The PR addresses the missing query parameters in the links section of the `getUsers()` API.

### CODE REVIEW

**Comment by george-haddad** [comment]
Maybe we also include a small note here saying that it will include the query parameters? 

https://github.com/acrolinx/platform-api/pull/100/files#diff-dc31b80e161d209b2d3d032acd88ad5047e9cabf6daf1142cecb4f87865b8d1aL1391

I am not sure if we should add it or not so referring to @madeline-brennan to see if we should add a note there or not. Personally I think fixing the example is ok as it mimics the query params shown.

**Review by george-haddad — APPROVED** [review]
Looks good to me.

**Comment by mrnyax** [comment]
> Maybe we also include a small note here saying that it will include the query parameters?
> 
> https://github.com/acrolinx/platform-api/pull/100/files#diff-dc31b80e161d209b2d3d032acd88ad5047e9cabf6daf1142cecb4f87865b8d1aL1391
> 
> I am not sure if we should add it or not so referring to @madeline-brennan to see if we should add a note there or not. Personally I think fixing the example is ok as it mimics the query params shown.

Added a small doc for this.

File: apiary.apib [hunk 1]
```diff
@@ -1388,7 +1388,7 @@ This is also the case for the values in the headers.
 This will also reflect any changes to the underlying data source. Say you're on page 2 of 10 with 10 records per page. 
 You then decide to delete 10 users before you call the API to fetch page 3. Now when you switch to page 3, the response will show page 3 of 9.
 
-The example response includes the pagination URLs in the `links` object so you know what to expect.
+The example response includes the pagination URLs in the `links` object so you know what to expect. It includes query parameters present in the request.
```
Comment (madeline-brennan) [inline apiary.apib:1391]:
```suggestion
The example response includes the pagination URLs and query parameters in the `links` object so you know what to expect. 
```

**Review by madeline-brennan — APPROVED** [review]

**Pull request merged**